autogen/docs/dictionary.html.mustache

```diff
@@ -5489,241 +6019,260 @@ show [count turtles in-radius 3] of patch 0 0
+          <span class="prim_example">other-end</span>
+          <img alt="Turtle Command" src="images/turtle.gif"/> <img alt="Link Command" src="images/link.gif"/>
+        </h4>
+        <p>
+          If run by a turtle, reports the turtle at the other end of the
+          asking link.
+        </p>
+        <p>
+          If run by a link, reports the turtle at the end of the link that
+          isn't the asking turtle.
+        </p>
+        <p>
+          These definitions are difficult to understand in the abstract, but
+          the following examples should help:
+        </p>
+        <pre>
 ask turtle 0 [ create-link-with turtle 1 ]
 ask turtle 0 [ ask link 0 1 [ show other-end ] ] ;; prints turtle 1
 ask turtle 1 [ ask link 0 1 [ show other-end ] ] ;; prints turtle 0
 ask link 0 1 [ ask turtle 0 [ show other-end ] ] ;; prints turtle 1
 </pre>
-      <p>
-        As these examples hopefully make plain, the &quot;other&quot; end
-        is the end that is neither asking nor being asked.
+        <p>
+          As these examples hopefully make plain, the &quot;other&quot; end
+          is the end that is neither asking nor being asked.
+        </p>
       </div>
-    <div class="dict_entry" id="out-link-neighbor">
-      <h3>
-        <a>out-&lt;breed&gt;-neighbor?</a>
-        <a>out-link-neighbor?<span class="since">4.0</span></a>
-      </h3>
-      <h4>
-        <span class="prim_example">out-&lt;breed&gt;-neighbor? <i>turtle</i></span>
-        <span class="prim_example">out-link-neighbor? <i>turtle</i></span>
-        <img alt="Turtle Command" src="images/turtle.gif">
-      </h4>
-      <p>
-        Reports true if there is a directed link going from the caller to
-        <i>turtle</i> or if there is an undirected link connecting the caller
-        with <i>turtle</i>. You can think of this as "can I get from the caller
-        to <i>turtle</i> using a link?"
-      <pre>
+      <div class="dict_entry" id="out-link-neighbor">
+        <h3>
+          <a>out-&lt;breed&gt;-neighbor?</a>
+          <a>out-link-neighbor?<span class="since">4.0</span></a>
+        </h3>
+        <h4>
+          <span class="prim_example">out-&lt;breed&gt;-neighbor? <i>turtle</i></span>
+          <span class="prim_example">out-link-neighbor? <i>turtle</i></span>
+          <img alt="Turtle Command" src="images/turtle.gif"/>
+        </h4>
+        <p>
+          Reports true if there is a directed link going from the caller to
+          <i>turtle</i> or if there is an undirected link connecting the caller
+          with <i>turtle</i>. You can think of this as "can I get from the caller
+          to <i>turtle</i> using a link?"
+        </p>
+        <pre>
 crt 2
 ask turtle 0 [
-  create-link-to turtle 1
-  show in-link-neighbor? turtle 1  ;; prints false
-  show out-link-neighbor? turtle 1 ;; prints true
+create-link-to turtle 1
+show in-link-neighbor? turtle 1  ;; prints false
+show out-link-neighbor? turtle 1 ;; prints true
 ]
```

Comment:
The update got rid of the extra spacing, but the primitives like `ask`, `if`, `while`, and `to` look like they've still lost their indentation.  Some of the comments, too.  They should keep the same spacing they had previously.